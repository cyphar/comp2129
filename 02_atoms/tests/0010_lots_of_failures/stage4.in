UNDO
UNDO 2
UNDO abc
START 3 2 10
UNDO
UNDO ab
UNDO   a    b 2-3
PLACE 0 0
UNDO
UNDO a a
UNDO
