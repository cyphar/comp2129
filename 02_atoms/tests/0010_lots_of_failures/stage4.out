Game Not In Progress

Game Not In Progress

Game Not In Progress

Game Ready
Red's Turn

Cannot Undo

Invalid Command

Invalid Command

Green's Turn

Red's Turn

Invalid Command

Cannot Undo

